index.js

```diff
@@ -3,26 +3,26 @@ const core = require('@actions/core');
 const { exec } = require('@actions/exec');
 const fs = require('fs');
 
-const scanScript = 'inline_scan';
-const defaultAnchoreVersion = '0.8.1';
+const grypeBinary = 'grype'
+const grypeVersion = '0.1.0-beta.11'
 
 // sarif code
 function convert_severity_to_acs_level(input_severity, severity_cutoff_param) {
-    var ret = "error"
-    const severityLevels = {
-    'Unknown': 0,
-    'Negligible': 1,
-    'Low': 2,
-    'Medium': 3,
-    'High': 4,
-    'Critical': 5
-    }
+  var ret = "error";
```

Comment:
`var` should be `let` here to avoid scope pollution

Reply:
(ditto this for all `var` uses)

Reply:
This PR didn't introduce this, as well as the other template literal changes, so I'm hesitant to make more changes even if they make sense. I do think that they can be improved/fixed though, so I would prefer to cut tickets to address those.

Reply:
Okay, thanks, that sounds good to me

Reply:
https://github.com/anchore/scan-action/issues/50